saleor/tax/utils.py

```diff
@@ -179,6 +179,11 @@ def normalize_tax_rate_for_db(tax_rate: Decimal) -> Decimal:
     return tax_rate / 100
```

Comment:
I know, that this is not a scope of this PR, but shouldn't we have a protection here for divide by zero?
something like:
```
if tax_rate == 0:
    return 0
```

Reply:
We're dividing by 100 always here, `0/100` is allowed ;)